game_of_life_review.py

```diff
@@ -0,0 +1,175 @@
+with open("entry.txt", "r") as f:
+    gen_num = int(f.readline())
+    field_size = [int(i) for i in f.readline().split(" ")
+                  ]  # 1 argument = y, 2 argument = x
+    first_gen = [[j for j in i.rstrip("\n")] for i in f.readlines()]
+
+
+def new_cell_state(field_state: list, cell_x: int, cell_y: int) -> str:
+    # Coordinates which are NOT placed on borders
+    if (cell_x > 0 and cell_x < (field_size[1]-1)) and (cell_y > 0 and cell_y < (field_size[0]-1)):
+        cells_neighbours = [
+            field_state[cell_x-1][cell_y-1],

```

Comment:
Wow! A lot of corner cases :) I would recommend making a method like `getActualCoords(x, y, sizeX, sizeY)` that is converting coordinates outside of the field to actual field coordinates.

For example, you have a board 3x3 (0:0 = top left corner):

```
getActualCoords(0, 0, 3, 3) => 0;0
getActualCoords(1, 2, 3, 3) => 1;2

getActualCoords(1, 0, 3, 3) => 2;0
getActualCoords(3, 0, 3, 3) => 0;0
```